src/connection.ts

```diff
@@ -1108,13 +1114,14 @@ export class Connection extends EventEmitter {
       throw new Error('Cannot determine exchange rate. Destination account is unknown')
     }
 
+    let retryDelay = RETRY_DELAY_START
     let testPacketAmounts = [1, 1000, 1000000, 1000000000, 1000000000000] // 1, 10^3, 10^6, 10^9, 10^12
     let attempts = 0
 
-    // set a max attempts in case F08 errors keep occurring
-    while (!this.exchangeRate && testPacketAmounts.length > 0 && attempts < 3) {
+    // set a max attempts in case F08 & TXX errors keep occurring
+    while (!this.exchangeRate && testPacketAmounts.length > 0 && attempts < 20) {
```

Comment:
This kind of option should either be configurable or up in a constant at the top of the file

Reply:
Good idea, to be clear we are talking about making attempts configurable?

Reply:
Yes, though in this case I think it's fine to put that as a constant instead of a config option

Reply:
Made it into a constant.